Mediapipe.Net.Examples.FaceMesh/mediapipe

```diff
@@ -0,0 +1 @@
+../mediapipe
```

Comment:
I wonder if we should keep that file to make it easier for other people.

How about adding a submodule to the Mediapipe repo itself?